Remove GCP Broker provider (#4858)
kyma-project/kyma: docs/migration-guides/1.3-1.4.md

# Migrate from v1.3 to v1.4

>**TIP:** To learn more about upgrading Kyma, read [this](https://kyma-project.io/docs/master/root/kyma/#installation-upgrade-kyma) document.

## Service Management

As the Google Cloud Platform (GCP) Service Broker is being deprecated, we have removed this addon. If you have already installed the GCP Service Broker in Kyma 1.3 you see this addon. 
If you still haven't installed any GCP services in your Namespace, delete the GCP Service Broker instance in this Namespace.

You can still manage the existing GCP services using Kyma until November 5, 2019. On or after that time, you will be able to manage services using the API, the command line or Google Cloud Platform Console.

> Consider removing GCP Service Bindings, Service Instances, and GCP Service Broker instances that you don't need. To do so, follow these steps:

Reviewer: ```suggestion
If you have installed the GCP Service Broker addon but haven't provisioned any of its services in your Namespace, delete the GCP Service Broker instance from the Namespace. If you have provisioned any GCP Service Broker services, consider removing the ones that you don't need. To do so, follow these steps:
```

1. Delete all bindings to GCP Service Instances.
2. Delete all GCP Service Instances.
3. Delete all GCP Service Broker instances.

> >**NOTE:** The charges incurred by the remaining resources created through the GCP Service Broker will continue to accrue. For more information, read the official Google Cloud [documentation](https://cloud.google.com/kubernetes-engine/docs/concepts/google-cloud-platform-service-broker).

Reviewer: ```suggestion
The charges incurred by the remaining resources created through the GCP Service Broker will continue to accrue. For more information, read the official Google Cloud [documentation](https://cloud.google.com/kubernetes-engine/docs/concepts/google-cloud-platform-service-broker).
```

# Migrate from v1.3 to v1.4

>**TIP:** To learn more about upgrading Kyma, read [this](https://kyma-project.io/docs/master/root/kyma/#installation-upgrade-kyma) document.

## Service Management
### Google Cloud Platform Service Broker addon

> As the Google Cloud Platform Service Broker is being [deprecated](https://cloud.google.com/kubernetes-engine/docs/concepts/google-cloud-platform-service-broker), we changed this addon to use different implementation. If you have already installed the Google Cloud Platform Service Broker in Kyma 1.3, you see this addon with two versions.

Reviewer: ```suggestion
As the Google Cloud Platform (GCP) Service Broker is being deprecated, we have changed this addon to use a different implementation. This is why you see this addon in two versions if you have already installed the GCP Service Broker in Kyma 1.3. 
```

> If you haven't installed any GCP services in a given Namespace, delete the Google Cloud Platform Service Broker instance in this Namespace.

Reviewer: ```suggestion
If you still haven't installed any GCP services in your Namespace, delete the GCP Service Broker instance in this Namespace.
```

Reviewer: Do we need this sentence? We state it again below in "Consider cleaning up...".

Author: I think we need this sentence.

You can still manage the existing GCP services using Kyma until November 5, 2019. On or after that time, you will be able to manage services using the API, the command line or Google Cloud Platform Console.

> Consider cleaning up not needed GCP Service Bindings, Service Instances and removal Google Cloud Platform Service Broker instances. Follow these steps:

Reviewer: ```suggestion
Consider removing GCP Service Bindings, Service Instances, and GCP Service Broker instances that you don't need. To do so, follow these steps:
```

1. Delete all bindings to GCP Service Instances.
2. Delete all GCP Service Instances.

> 3. Delete all Google Cloud Platform Service Broker Service Instances.

Reviewer: ```suggestion
3. Delete all GCP Service Broker instances.
```

> If you need a new instance of GCP services, provision Google Cloud Platform Service Broker 0.0.2 and use its service classes.

Reviewer: ```suggestion
If you need a new instance of GCP services, provision GCP Service Broker 0.0.2 and use its Service Classes.
```

Reviewer: ?
